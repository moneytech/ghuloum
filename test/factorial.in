(labels
    ((fact
        (code (n)
            (if (zero? n)
                1
                (* n (labelcall fact (sub1 n )))))))
    (labelcall fact 5)
)
