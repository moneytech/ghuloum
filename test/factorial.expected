0x1e0
